### PR TITLE
a litte better info and checking

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

File: scripts/loadup-init.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+o#!/bin/sh
```
Comment (nbriggs) [inline scripts/loadup-init.sh:1]: Looks like a typo included here

File: run-medley [hunk 2]
```diff
@@ -172,9 +172,15 @@ export PATH="$MAIKODIR"/`osversion`.`machinetype`:"$oldpath"
 
 cd "$OLDPWD"
 
-echo "sysout is $LDESRCESYSOUT"
-echo "running `which $prog` $geometry $screensize $mem"
-echo "start $LDEINIT"
+whichprog=`which $prog`
+
+if [ -z "$whichprog" ] ; then
```
Comment (nbriggs) [inline run-medley:177]: `which` isn't the best choice here -- it's a c-shell-ism. One alternative is
```
if command -v $prog > /dev/null 2>&1; then
    # we're good
else
   # we're not
fi
```

Comment (masinter) [reply run-medley:177]: trying to fix #271

Comment (masinter) [reply run-medley:177]: I don't know what you mean by command? Not part of my Linux.

Comment (nbriggs) [reply run-medley:177]: It's literally the word "command"

**Comment by nbriggs** [comment]
Are you going to roll this branch back to the 0ba2ce (before the mistaken commit of the GETHOSTINFO change) and force push it so that we can approve/merge it?

**Comment by masinter** [comment]
I am confused by the git status; not sure what 'force push' means in this context

**Comment by nbriggs** [comment]
You have a branch... named script-tweaks that needs to be adjusted before the pull request should be done.  You check out the branch, then checkout the commit before the last, then "git push --force" which will update github with the branch rolled back to the place you had checked out.   Then the PR can be done and it will only include the branch in its new fixed state.   If you don't use "--force" it will not let you push the branch because it would not be simply adding commits.  Never use "--force" on "master" or on any branch that has already been merged anywhere.

**Pull request closed without merging**